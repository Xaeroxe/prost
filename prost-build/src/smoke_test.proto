syntax = "proto2";

package smoke_test;

message SmokeRequest {
}

message SmokeResponse {
}

// Just a smoke test service.
//
// ```
// It'd be a mistake to try to rust doc test this.
// ```
//
//     Here's another code block, just for kicks.
// This isn't a code block.
//
//     And another, just to be sure.
service SmokeService {

    // A detached comment block.

    // Blow some smoke.
    rpc BlowSmoke(SmokeRequest) returns (SmokeResponse);
}
