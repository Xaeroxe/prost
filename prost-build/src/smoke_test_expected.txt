#[derive(Clone, PartialEq, ::prost::Message)]
pub struct SmokeRequest {
}
#[derive(Clone, PartialEq, ::prost::Message)]
pub struct SmokeResponse {
}
/// Just a smoke test service.
///
/// ``` text
/// It'd be a mistake to try to rust doc test this.
/// ```
///
/// ``` text
/// Here's another code block, just for kicks.
/// ```
///
/// This isn't a code block.
///
/// ``` text
/// And another, just to be sure.
/// ```
trait SmokeService {
    // A detached comment block.

    /// Blow some smoke.
    fn blow_smoke(SmokeRequest) -> SmokeResponse;
}
pub mod utils { }
